Screens/Home/HomeScreen.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { GetArticles } from "../../Services/ArticlesApiService";
+import { StyleSheet, Text, View, FlatList } from "react-native";
+import { Image } from "react-native-elements";
+
+export default class HomeScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      articles: []
+    };
+  }
+
+  componentDidMount() {
+    GetArticles().then(res => {
+      this.updateArticlesStateHandler(res);
+    });
+  }
+
+  updateArticlesStateHandler(articles) {
+    this.setState({
+      articles: articles
+    });
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <FlatList
+          data={this.state.articles}
+          renderItem={this.renderArticles.bind(this)}
+          keyExtractor={item => item.id.toString()}
+        />
+      </View>
+    );
+  }
+
+  renderArticles({ item }) {
+    const article = item;
+    return (
+      <View>
+        <Image
+          style={{ width: 100, height: 100 }}
+          source={{ uri: article.image }}
+        />
+        <Text>{article.category}</Text>
```

Comment:
In your LOFIs, do you have the category above the title of the article?

Reply:
No the category should be "invisible" and only be shown in the dropdown menu we are implementing later on.